cli/lib/lod.js

```diff
@@ -4,12 +4,14 @@ import { getAverageColor } from "fast-average-color-node";
 import simplify from "./simplification/index.js";
 import { prepareData } from "./simplification/prepare-data.js";
 import * as print from "./helper/print.js";
+import { getFolderPath, copyFolder } from "./helper/files.js";
 
 const io = new NodeIO();
 
 export const copyOriginalArtifact = (pathName, file) => {
```

Comment:
I suggest to extract this from the gltf-transform scripts into the `files.js` helpers as there are no glTF steps involved in copying folders.